### DISCUSSION TITLE
Return value must be of type Filament\Support\Enums\Alignment|string, resource returned

### DISCUSSION THREAD

**Discussion by Kovah** [awcodes/filament-curator#332] [Q&A] [answered]
The Curator overview is loading fine, but when opening any other page like "Create" or editing media files, the app crashes with the following error:

```
Filament\Pages\BasePage::getFormActionsAlignment(): Return value must be of type Filament\Support\Enums\Alignment|string, resource returned 
(View: /app/vendor/filament/filament/resources/views/components/form/actions.blade.php) 
(View: /app/vendor/filament/filament/resources/views/components/form/actions.blade.php)
```

This started to happen after upgrading Curator and Filament via `composer update`:

```
  - Upgrading awcodes/filament-curator (v3.0.5 => v3.0.7)
  - Upgrading filament/actions (v3.0.73 => v3.0.83)
  - Upgrading filament/filament (v3.0.73 => v3.0.83)
  - Upgrading filament/forms (v3.0.73 => v3.0.83)
  - Upgrading filament/infolists (v3.0.73 => v3.0.83)
  - Upgrading filament/notifications (v3.0.73 => v3.0.83)
  - Upgrading filament/spatie-laravel-settings-plugin (v3.0.73 => v3.0.83)
  - Upgrading filament/support (v3.0.73 => v3.0.83)
  - Upgrading filament/tables (v3.0.73 => v3.0.83)
  - Upgrading filament/widgets (v3.0.73 => v3.0.83)
```

After that, I re-created the Curator config, which is the standard one, except the path and storage disk are configured. In the Filament panel I specified like in the docs, but without any specific configuration:

```
  ->plugins([
      CuratorPlugin::make(),
  ])
```

I am not sure if this is caused by something about my specific setup. Is anyone else experiencing those issues? I basically can't use Curator anymore. :/

**Comment by awcodes** [comment, marked as answer]
Did you publish the filament views?

**Reply by Kovah** [reply]
Not sure what that means, I'm pretty new to Filament. I only run `filament:upgrade` during the deployment so that the assets are there.

**Reply by awcodes** [reply]
What i'm getting at is this path doesn't look right
'/app/vendor/filament/filament/resources/views/components/form/actions.blade.php'

**Reply by awcodes** [reply]
yea, I'm not having any issues on 3.0.83.

make sure you are running 'php artisan filament:upgrade' after you update filament.

**Reply by Kovah** [reply]
It seems there's something odd with the cached views of filament, it also happens with other plugins and forms. Sorry for the inconvenience.
